### PR TITLE
complete ticket 8 with the corresponding array of articles objects.

### PR DESCRIPTION
**Pull request by m-mohamed22**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -41,6 +41,49 @@ describe("1. GET/api/topics", () => {
...
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
```
Comment (Augs0) [inline __tests__/app.test.js]: Double check the ticket requirements. We don't want a body here. If you imagine it as a blog or news site, this is like the list of titles we can click :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -147,16 +189,14 @@ describe("3: PATCH /api/articles/:article_id", () => {
...
-    return (
-      request(app)
-        .patch("/api/articles/888")
-        // .send(requestBody)
-        .expect(404)
-        .then(({ body }) => {
-          expect(body.msg).toBe("Article ID 888 does not exist");
-        })
-    );
+    const requestBody = { inc_votes: 0 };
```
Comment (Augs0) [inline __tests__/app.test.js:191]: Please avoid working on other tickets at the same time. This should have been handled in the patch ticket

**Review by Augs0 — CHANGES_REQUESTED** [review]
Almost there! Just make sure body isn't coming back on the article object

**Review by emilyb93 — APPROVED** [review]
Brill great work so far!

**Pull request merged**